Tensile/Components/LraTileAssignment.py

```diff
@@ -91,9 +91,10 @@ def __call__(self, writer, kernel, tP):
         wReg    = writer.vgprPool.checkOut(1,"wReg") # quotient
         tReg    = writer.vgprPool.checkOut(1,"tReg") # remainder
         kReg    = writer.vgprPool.checkOut(1,"kReg") # remainder
+        mReg    = writer.vgprPool.checkOut(1,"mReg") # remainder
```

Comment:
Can we limit this vreg allocations only when ThreadSeparateGlobalRead is true?
It can increase total amount of vreg usage and it can trigger vreg overflow.

Reply:
By the way, is the comment "reminder" here correct?